src/app/static/src/tests/components/runReport/parameterList.test.ts

```diff
@@ -53,8 +53,9 @@ describe(`run report parameter list`, () => {
                 propsData: {params: innerParams}
             }
         )
-        wrapper.vm.$emit("input")
-        expect(wrapper.emitted().paramsChanged.length).toBe(1)
+        const inputs = wrapper.find("table").findAll("input")
+        await inputs.at(0).setValue("test Value1")
+        expect(wrapper.emitted().paramsChanged.length).toBe(2)
```

Comment:
Want to check the values were correctly updated too in the next line? (Check `paramsChanged[1]` not `paramsChanged[0]`)